components/demo/demo-snippet.js

```diff
@@ -186,6 +202,12 @@ class DemoSnippet extends LitElement {
 		this._applyAttr('dir', this._dir, true);
 	}
 
+	_handleFullscreenViewChange(e) {
+		this.fullscreenView = e.target.on;
+		const event = new CustomEvent('fullscreen-toggle', { bubbles: true, composed: true });
```

Comment:
I think this should follow our event naming conventions and be something like `d2l-demo-snippet-fullscreen-toggle`.